telecine/src/main/java/com/jakewharton/telecine/TelecineService.java

```diff
@@ -35,6 +37,10 @@ public static Intent newIntent(Context context, int resultCode, Intent data) {
 
   private final RecordingSession.Listener listener = new RecordingSession.Listener() {
     @Override public void onStart() {
+      if(showTouchesProvider.get()) {
+        Settings.System.putInt(TelecineService.this.getContentResolver(), "show_touches", 1);
```

Comment:
I really don't like using `Telecine.this` reference but this location seemed apropriate for this case. If you have another sugestion, i would gladly change it.